assets/src/application/ui/forms/ee-form/config.tsx

```diff
@@ -0,0 +1,233 @@
+import { EspressoFormProps } from './types';
+
+export const formConfig: EspressoFormProps = {
+	onSubmit: (values) => console.log(values),
+	initialValues: {},
```

Comment:
so will this have to consist of the same structure as the "sections" field below?
if so, then would it not be easier to simply add an "initialValue" prop to each of the fields within "sections"?

Reply:
Yes, if you pass `initialValues` prop to form, the value has to be in the same shape as the form fields but you can also pass `initialValue` prop to every individual field.